tutorials/plot_forward.py

```diff
@@ -86,15 +86,26 @@
                        surfaces='head-dense')
 
 ###############################################################################
+# .. _plot_forward_source_space:
+#
 # Compute Source Space
 # --------------------
 #
-# The source space defines the position of the candidate source locations.
-# The following code compute such a cortical source space with
-# an OCT-6 resolution.
+# The source space defines the position and orientation of the candidate source
+# locations. There are two types of source spaces:
+# - **source-based** source space when the candidates are confined to a surface.
```

Comment:
It might not clear to me what is this `surface`. Where can we point?

Reply:
Typically it's the `white` Freesurfer surface, defined by the boundary between the cortical gray and white matter. This is the `surface` argument in `setup_source_space`.

Reply:
(we don't have good docs for this yet, it's a documentation TODO)

Reply:
> (we don't have good docs for this yet, it's a documentation TODO)

is there an issue open so that we can add a note to link back here?

Reply:
Yes we should have something like this but I don't think we have it yet

Reply:
See #2426 which was rolled into #1495

For other docs things, see also #3985

Reply:
wooo I was not aware of these issues.